README.md

```diff
@@ -50,13 +56,12 @@ Secondly, balenaSound will play connection/disconnection notification sounds at
 
 **Note:** these variables should be defined as integer values without the `%` symbol.
 
-### Deploy this application
+### Set bluetooth pin code
+
+By default, balenaSound bluetooth will connect using Secure Simple Pairing mode with no pin code required. If you would like to override this, you can define the `BLUETOOTH_PIN_CODE` environment variable with the pin code you desire.
```

Comment:
Can we indicate to the user what format of pin code is required? I.e. 4 digit numeric (or whatever it happens to be)

Reply:
@tmigone you can copy and Paste this: `**Note:** You can use up to 16 character as the pin code. They can be alphanumeric (A-Z, a-z, 1-9, !%§, etc.).` it is from my closed PR.

Reply:
And if you want can you use this picture:
![pin-code-config](https://user-images.githubusercontent.com/40795980/68235378-6eadb080-0003-11ea-9a1f-db03f21989be.png)

Reply:
@chrisys Updated readme to specify pin code format and also added code to check if pin is valid. 
@AlexProgrammerDE you are correct that pin code can be almost anything up to 16 character's, but i've seen some devices that only allow numeric input, so I limited the PIN code to what the bluetooth spec says.

Reply:
does this actually work? I have set this as an environnment variable in Balena Dashboard but even after a reboot and a restart I can still connect to the device without entering the PIN.